changelog/137.breaking.rst

```diff
@@ -0,0 +1 @@
+Rename of `sunxspex_fitting` module tp `fitting_legacy` and added new `fitting` module.
```

Comment:
```suggestion
Rename of `sunxspex_fitting` module to `fitting_legacy` and added new `fitting` module.
```

Reply:
Also the new fitter module hasn't been added which is fine but if not going to do in this PR then probably remove from change log.

Reply:
I didn't realise empty directories did not get added. I have now managed to get the `fitting` module to be included.